includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1322,10 +1340,16 @@ public function wc_setup_payment() {
 	public function wc_setup_payment_save() {
 		check_admin_referer( 'wc-setup' );
 
-		// Install WooCommerce Services with Stripe to enable deferred account creation.
 		if (
-			! empty( $_POST['wc-wizard-service-stripe-enabled'] ) &&
-			! empty( $_POST['stripe_create_account'] )
+			(
+				// Install WooCommerce Services with Stripe to enable deferred account creation
```

Comment:
These two comments here need a period at the end. ([ref](https://travis-ci.org/woocommerce/woocommerce/jobs/312442467)) Don't worry about the phpcs stuff that isn't from this pull request.